state/core_access.go

```diff
@@ -136,7 +133,6 @@ func NewCoreAccessor(
 		addr:     addr,
 		getter:   getter,
 		coreIP:   coreIP,
```

Comment:
Can't core be accesses by URL? Does it have to be by IP? I wonder how this would work with TLS (where certs will be tied to a domain / URL).